content/components/confirmation-dialog.mdx

```diff
@@ -0,0 +1,207 @@
+---
+title: ConfirmationDialog
+description:
+  ConfirmationDialog is a specialized dialog component used to confirm user actions. It provides a rigid two-button
+  interface with a cancel and confirm action.
+reactId: confirmation-dialog
+figmaId: confirmation-dialog
```

Comment:
❓ Question for reviewers: how do I validate these IDs are pointing to the correct reference outbound to the React source code or Figma URL?